src/main/java/core/mate/academy/service/BulldozerProducer.java

```diff
@@ -0,0 +1,27 @@
+package core.mate.academy.service;
+
+import core.mate.academy.model.Bulldozer;
+import core.mate.academy.random.Color;
+import core.mate.academy.random.Model;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Random;
+
+public class BulldozerProducer implements MachineProducer<Bulldozer> {
+    private static final int NUMBER_OF_MACHINES = 3;
+    private Random random = new Random();
+
+    @Override
+    public List<Bulldozer> get() {
+        List<Bulldozer> bulldozers = new ArrayList<>();
+
+        for (int i = 0; i < NUMBER_OF_MACHINES; i++) {
+            String randomName = Model.values()[random.nextInt(Model.values().length)].name();
+            String randomColor = Color.values()[random.nextInt(Color.values().length)].name();
```

Comment:
Minor comment for this task, but pay attention to it: you wrote these 2 lines thrice in your solution, try avoiding code duplication in your solution